app/controllers/events_controller.rb

```diff
@@ -35,6 +35,8 @@ def invalid_request(message = 'invalid request', type: 'invalid_request')
   end
 
   def verify_signature
+    return true if Rails.env.development?
```

Comment:
I meant whether this method will ever be tested for when the environment is not develop, but I see the unit tests now :)